src/components/pages/home.jsx

```diff
@@ -1,45 +1,59 @@
 import {Link} from "react-router"
-import React from "react"
+import React, {PropTypes} from "react"
 
 import BreadCrumb from "../breadcrumb"
+import ExternalLink from "../external-link"
 
 
 const HomePage = React.createClass({
+  propTypes: {
+    countryPackageName: PropTypes.string.isRequired,
```

Comment:
Just so that I understand : where will the value of these two properties will come from ? Which part of the code gets it from the openfisca web api ?